File: 2271-rearrange-array-elements-by-sign/rearrange-array-elements-by-sign.cpp
class Solution {
public:
    vector<int> rearrangeArray(vector<int>& nums) {
        vector<int>p;
        vector<int>m;
        vector<int>ans;
        for(int i=0;i<nums.size();i++)
        {
            if(nums[i]>0)
            {
                p.push_back(nums[i]);
            }
            else
            {
                m.push_back(nums[i]);
            }
        }
        for(int j=0;j<p.size();j++)
        {
            ans.push_back(p[j]);
            ans.push_back(m[j]);
        }
        return ans;
    }
};

File: 2271-rearrange-array-elements-by-sign/rearrange-array-elements-by-sign.py
class Solution:
    def rearrangeArray(self, nums: List[int]) -> List[int]:
        p=[]
        m=[]
        for i in range(len(nums)):
            if(nums[i]>0):
                p.append(nums[i])
            else:
                m.append(nums[i])
        ans=[]
        for i in range(len(p)):
            ans.append(p[i])
            ans.append(m[i])
        return ans
        


        
        